CI/travis/before_deploy

```diff
@@ -18,8 +18,41 @@ else
 fi
 pwd
 
+if [ -z "${LDIST}" -a -f "build/.LDIST" ] ; then
+	LDIST=$(cat build/.LDIST)
+fi
 if [ -z "${LDIST}" ] ; then
-	LDIST=-$(lsb_release -c | awk '{print $NF}')
+	# provide something like "-Debian-buster-amd64" or
+	# "-Ubuntu-xenial-amd64" or "-CentOS-7.5-x86_64"
+	case "$(uname)" in
+	Linux*)
+		LDIST=$(awk -F= '/^NAME/{print $2}' /etc/os-release | \
+			sed -e 's| GNU/Linux||' -e 's/ Linux//' -e 's/"//g' \
+			-e 's/ /-/g' -e 's|/|_|g')
+		case "${LDIST}" in
+		Debian|Raspbian|Linaro|Ubuntu)
+			LDIST=-${LDIST}-$(lsb_release -c | awk '{print $NF}')
+			LDIST=${LDIST}-$(dpkg --print-architecture)
```

Comment:
this means download links should change, right ?
System architecture wasn't part of LDIST before